test/definitions/capacitance_from_charge_and_voltage_test.py

```diff
@@ -0,0 +1,55 @@
+# Description
+## Assume we have a capacitor with 6 Coulombs of charge in it and having 3 volts on it's terminals.
+## According to the definition of Capacitance this capacitor should have C = 2 Farads.
+
+from collections import namedtuple
+from pytest import approx, fixture, raises
+
+from symplyphysics import (
+    units, convert_to, SI, errors, solve
+)
+from symplyphysics.definitions import capacitance_from_charge_and_voltage as capacitance_def
+
+@fixture
+def test_args():
+    C = units.Quantity('C')
+    SI.set_quantity_dimension(C, units.capacitance)    
+    SI.set_quantity_scale_factor(C, 2 * units.farad)
+    Q = units.Quantity('Q')
+    SI.set_quantity_dimension(Q, units.charge)
+    SI.set_quantity_scale_factor(Q, 6 * units.coulomb)
+    U = units.Quantity('U')
+    SI.set_quantity_dimension(U, units.voltage)
+    SI.set_quantity_scale_factor(U, 3 * units.volts)
+
+    Args = namedtuple('Args', ['C', 'Q', 'U'])
+    return Args(C = C, Q = Q, U = U)
+
+def test_basic_capacitance(test_args):
```

Comment:
You should have the same set of tests for each of the `calculation` functions. So, `test_basic_charge`, `test_bad_capacitance`, `test_bad_voltage_for_charge`, and so on.